app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Auth;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Question;
+use App\User;
+use App\Category;
+use App\Choices;
+
+class UserController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('check', ['only' => ['followUser','unFollowUser']]);
+    }
+
+    public function displayUsers()
+    {
+        $users = User::all();
+
+        return view('users.DispUsers',compact('users')); 
+    }
+    
+    public function show(int $userId)
```

Comment:
You can create a `User ` instance as a parameter on `show()` function and you can directly access the followers and following relatinship